rentals.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Please use a getter and setter method for book and person also. In other words, kindly add them to `` attr_accessor``